Impala hook implention

blcksrx opened the pull request:
Impala hook implention
related:  #8918

mik-laj commented on airflow/providers/apache/impala/README.md:
```diff
@@ -0,0 +1,18 @@
+ under the License.
+ -->
+
+# Package apache-airflow-backport-providers-apache-impala
```
Can you add provider.yaml also?

blcksrx replied:
Yes. I did it

Commit:
Impala hook implention

Commit:
Impala hook implention

mik-laj commented on airflow/providers/apache/impala/provider.yaml:
```diff
@@ -0,0 +1,33 @@
+
+---
+package-name: apache-airflow-providers-apache-impala
+description: |
```
```suggestion
package-name: apache-airflow-providers-apache-impala
name: Apache Impala
description: |
```

mik-laj commented on docs/apache-airflow-providers-apache-impala/index.rst:
```diff
@@ -0,0 +1,29 @@
+    under the License.
+
+``apache-airflow-providers-apache-impala``
+=========================================
```
```suggestion
==========================================
```

blcksrx replied:
Thanks! there is another error related to spelling error.

```
#################### Spelling errors summary ####################
==================== apache-airflow-providers-apache-impala ====================
```

what about it?

mik-laj committed:
Move operator guides to provider documentation packages (#12681)

ashb committed:
Remove deprecated dagbag metrics (#12695)

These were deprecated in 1.10.6 via #6157, so we should remove them
before 2.0 rolls around.

Hasan-J committed:
Deprecate BaseHook.get_connections method (#10135) (#10192)

Co-authored-by: Ash Berlin-Taylor <ash_github@firemirror.com>

mpolatcan committed:
Add Getir to in the wild! (#12719)

* Add Getir to in the wild!

* hotfix - Add Getir to in the wild

mik-laj committed:
Move apache-airflow docs to subdirectory (#12715)

potiuk committed:
Output of installing remaining packages is shown also on success (#12723)

Previously the output of instaling remaining packges when testing
provider imports was only shown on error. However it is useful
to know what's going on even if it clutters the log.

Note that this installation is only needed until we include
apache-beam in the installed packages on CI.

Related to #12703

This PR shows the output always .

potiuk committed:
Improve verification of images with PIP check (#12718)

Verification of images with PIP is done in separate jobs and
they provide useful information to committers and contributors
when the pip check fails.

ryw committed:
Stronger language re: SQLite (#12727)

mik-laj committed:
Fix static checks - #12715 (#12729)

kaxil committed:
Allow switching xcom_pickling to JSON/Pickle (#12724)

Without this commit, the Webserver throws an error when
enabling xcom_pickling in the airflow_config by setting `enable_xcom_pickling = True`
(the default is `False`).

Example error:

```
>           return pickle.loads(result.value)
E           _pickle.UnpicklingError: invalid load key, '{'.

airflow/models/xcom.py:250: UnpicklingError
--------------------------------------------------
```

potiuk committed:
Pins PIP to 20.2.4 in our Dockerfiles (#12738)

Until we make sure that the new resolver in PIP 20.3 works
we should pin PIP to 20.2.4.

This is hopefully a temporary measure.

Part of #12737

potiuk committed:
User-friendly output of Breeze and CI scripts (#12735)

Commit:
Fix chart jobs delete policy for improved idempotency (#12646)

The chart has two jobs (migrate-database & create-user).
These jobs are run post-install and post-upgrade and only deleted on success.

So if for some reason (quick reinstall / upgrade), the job fails or is stuck then helm
will fail because the job already exists.

This commit sets the `helm.sh/hook-delete-policy` to `before-hook-creation,hook-succeeded`
so helm will always delete the jobs before creating them again.

shivanshs9 committed:
Added Headout to INTHEWILD (#12734)

Signed-off-by: Shivansh Saini <shivanshs9@gmail.com>

Commit:
Merge branch 'impala' of github.com:blcksrx/airflow into impala
